po/hr.po

```diff
@@ -310,10 +308,9 @@ msgstr "Otvori datoteku"
 #. TRANSLATORS: Add your name/nickname here
 #: src/window.py:284 src/window.py:285
 msgid "translator-credits"
-msgstr ""
+msgstr "sheep"
 
 #: src/ui/albumcoverbutton.ui:73
-#, fuzzy
 msgid "Album cover"
 msgstr "Album"
```

Comment:
You might've forgotten to translate "Album cover"

Reply:
Fixed!